### ISSUE TITLE
reCAPTCHA Size issue

### ISSUE THREAD

**Issue by JustJarethB** [putyourlightson/craft-campaign#85]
When enabling reCAPTCHA it seems only the invisible option works. Choosing normal results in the following JS Error:
```
Uncaught Error: grecaptcha.execute only works with invisible reCAPTCHA.
```

**Comment by putyourlightson** [comment]
Thanks, we'll look into this and let you know what we find.

**Comment by putyourlightson** [comment]
Fixed in version 1.11.0.
